### PR TITLE
Migrate bonny to latest master

### PR DESCRIPTION
**Pull request by mruoss**
Hi @sleipnir 

As promised I'd like to help you migrate to the latest version. I have removed a lot of code from `SpawnOperator` because:

* Logging will be done by `Bonny.Pluggable.Logger` (added to operator)
* Applying descendants will be done by `Bonny.Pluggable.ApplyDescendants` (added to operator)
* Other code was moved to the handler modules.

I have successfully started the operator but not tested it (i.e. I have not created custom resources)

### CODE REVIEW

**Comment by sleipnir** [comment]
Hi @mruoss 

Thanks. I greatly appreciate your help. I'll take a look at it calmly and I already approve the PR.
I am very pleased with your contribution.

File: spawn_operator/spawn_operator/lib/spawn_operator/controller/actor_host_controller.ex [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-    for_resource:
-      Bonny.API.CRD.build_for_controller!(
-        group: "spawn-eigr.io",
-        scope: :Cluster,
-        versions: [SpawnOperator.Versions.Api.V1.ActorHost]
-      )
+  use Bonny.ControllerV2
 
-  use SpawnOperator.Handler.ActorHostHandler
+  # step Bonny.Pluggable.SkipObservedGenerations
```
Comment (mruoss) [inline spawn_operator/spawn_operator/lib/spawn_operator/controller/actor_host_controller.ex:6]: Feel free to uncomment these steps. Note however, that you have to uncomment the according lines in the version modules as well.

**Comment by sleipnir** [comment]
@mruoss As we talked about, there are still some problems but I'll continue with the merge

**Comment by sleipnir** [comment]
@mruoss Thanks for the great contribution. Do you let me know when you can minimize that impact of events?

**Comment by mruoss** [comment]
> Do you let me know when you can minimize that impact of events?

Done.

**Pull request merged**